src/Apps/Artwork/Components/__tests__/ArtworkSidebar/ArtworkSidebarMetadata.test.tsx

```diff
@@ -0,0 +1,143 @@
+import { cloneDeep } from "lodash"
+import { graphql } from "react-relay"
+import { renderRelayTree } from "../../../../../DevTools"
+import {
+  EmptyMetadataNoEditions,
+  FilledOutMetadataMultipleEditionSets,
+  FilledOutMetadataNoEditions,
+  FilledOutMetadataOneEditionSet,
+  MetadataForAuctionWork,
+} from "../../../../__test__/Fixtures/Artwork/ArtworkSidebar/ArtworkSidebarMetadata"
+import { ArtworkSidebarClassification } from "../../ArtworkSidebar/ArtworkSidebarClassification"
+import { ArtworkSidebarMetadataFragmentContainer } from "../../ArtworkSidebar/ArtworkSidebarMetadata"
+import { ArtworkSidebarSizeInfo } from "../../ArtworkSidebar/ArtworkSidebarSizeInfo"
+
+jest.unmock("react-relay")
+
+describe("ArtworkSidebarMetadata", () => {
+  const data = null
+  const getWrapper = async (response = FilledOutMetadataNoEditions) => {
+    return await renderRelayTree({
+      Component: ArtworkSidebarMetadataFragmentContainer,
+      query: graphql`
+        query ArtworkSidebarMetadata_Test_Query {
+          artwork(id: "josef-albers-homage-to-the-square-85") {
+            ...ArtworkSidebarMetadata_artwork
+          }
+        }
+      `,
+      mockResolvers: {
+        Artwork: () => response,
+      },
+    })
+  }
+  describe("for non editioned artwork", () => {
+    it("displays title and year", async () => {
+      const wrapper = await getWrapper()
```

Comment:
Seeing as all the wrappers are the same inside these describe sections, you could define the wrapper once from a beforeAll function and then all the tests don’t even need to be async.

Probably reduces some noise and a slightly faster pattern, performance wise (negligible on its own, but noticeable when a test suite gets large).

Reply:
Ohh great idea. Also eliminates the annoying missing `async / await` thing that constantly arises.